test/services/childcare/send_docusign_envelope_test.rb

```diff
@@ -0,0 +1,45 @@
+require 'test_helper'
+
+class Childcare::SendDocusignEnvelopeTest < ServiceTestCase
+  
+  def setup
+    VCR.turn_on!
+    @family = create :family
+    @attendee = create :attendee, family: @family, first_name: 'Test', last_name: 'Recipient', email: 'test@example.com'
+    @child = create :child, family: @family
+    @family.primary_person = @attendee
+    @family.save
+  end
+
+  def teardown
+    VCR.turn_off!
+  end
```

Comment:
If we need to turn off VCR after every test then what do you think about doing that in a more global teardown callback in the test helper? This would prevent an issue from accidentally forgetting to turn it off.

Reply:
🏅

Reply:
Removed all that confusing stuff, you don't need to be turning it on/off like this.